image/bif.ksy

```diff
@@ -0,0 +1,67 @@
+meta:
+  id: bif
+  file-extension: bif
+  endian: le
+  title: Roku Base Index Frames (BIF) image archive format
+  license: MIT
+seq:
+  - id: header
+    type: header
+    doc: |
+      BIF header, containing magic, version, number of images,
+      timestamp multiplier, and reserved bytes
+  - id: images
+    type: image_data(_index)
+    repeat: expr
+    repeat-expr: header.num_images
+    doc: |
+      Image data section, containing timestamps, offsets,
+      and JPEG data of each image
+types:
+  header:
+    seq:
+      - id: magic
+        contents: [0x89, 0x42, 0x49, 0x46, 0x0d, 0x0a, 0x1a, 0x0a]
+        doc: |
+          This is a file identifier.
+          It contains enough information to identify the file type uniquely.
+      - id: version
+        type: u4
+        doc: |
+          This space is reserved for a revision number.
+          The current specification is file format version 0.
+      - id: num_images
+        type: u4
+        doc: |
+          This is an unsigned 32-bit value (N) that represents the
+          number of BIF images in the file. The number of entries
+          in the index will be N+1, including the end-of-data entry.
+      - id: timestamp_multiplier
+        type: u4
+        doc: |
+          This specifies the denomination of the frame timestamp values.
```

Comment:
Proposal: remove this doc and just encode its meanjng into an instance within each `image_data`